views/layouts/main.handlebars

```diff
@@ -20,49 +20,79 @@
   <link href="https://fonts.googleapis.com/css?family=Merriweather:300,300i,400,400i,700,700i,900,900i"
     rel="stylesheet">
   <link href="vendor/fontawesome-free/css/all.min.css" rel="stylesheet" type="text/css">
-
   <!-- Custom styles for this template -->
   <link href="css/coming-soon.min.css" rel="stylesheet">
-    <style>
-/* unvisited link */
-a:link {
-  color: white;
-}
-
-/* visited link */
-a:visited {
-  color: white;
-}
-
-/* mouse over link */
-a:hover {
-  color: white;
-}
-
-/* selected link */
-a:active {
-  color: white;
-}
-</style>
+  <style>
+    /* unvisited link */
+    a:link {
+      color: white;
+    }
+
+    /* visited link */
+    a:visited {
+      color: white;
+    }
+
+    /* mouse over link */
+    a:hover {
+      color: white;
+    }
+
+    /* selected link */
+    a:active {
+      color: white;
+    }
+
+    header {
+      padding: 156px 0 100px;
+    }
+
+    section {
+      padding: 150px 0;
+    }
+
+    .invalid-feedback {
+      font-size: 100%;
+      color: white;
+    }
+  </style>
   <script>
```

Comment:
Can the script live in e.g. `coming-soon.js` so it's not part of the HTML?

Reply:
I tried, gulp is not supporting es6.